RGB4R/Script.cs

```diff
@@ -87,56 +96,73 @@ private void OnTick(object sender, EventArgs e)
         }
 
         Chroma.PerformHeartbeat();
+
+        if (effectReserveCounter != 0)
+        {
+            effectReserveCounter--;
```

Comment:
This is measuring ticks/frames, not time.

If your game is running at 144 FPS, it would take around 2.9 seconds, but if is running at 60, it will take around 7.2 seconds.

Reply:
Should all be fixed now, thanks for the feedback.